ui/app/AppLayouts/Chat/views/communities/CommunityNewPermissionView.qml

```diff
@@ -381,11 +383,104 @@ StatusScrollView {
             color: Style.current.separator
         }
         StatusItemSelector {
+            id: inSelector
+
             Layout.fillWidth: true
             icon: Style.svg("create-category")
             iconSize: 24
             title: qsTr("In")
             defaultItemText: qsTr("Example: `#general` channel")
+
+            useLetterIdenticons: !wholeCommunitySelected || !inDropdown.communityImage
+
+            property bool wholeCommunitySelected: false
+
+            ListModel {
+                id: inModelCommunity
+
+                readonly property string colorWorkaround: inDropdown.communityData.color
+
+                Component.onCompleted: {
+                    append({
+                        imageSource: inDropdown.communityData.image,
+                        text: inDropdown.communityData.name,
+                        operator: OperatorsUtils.Operators.None
+                    })
+
+                    setProperty(0, "color", colorWorkaround)
+                }
+            }
+
+            ListModel {
+                id: inModelChannels
+            }
+
+            InDropdown {
+                id: inDropdown
+
+                model: root.rootStore.chatCommunitySectionModule.model
```

Comment:
Since it is new code, could we start here avoiding this type of calls and create the specific API in `rootStore`? Or maybe we can just use `CommunityStore` and add the needed variables there.

Reply:
Currently, I'm using the model from place where it's currently defined and used by various components. Changing that would require bigger refactor. So in general it's needed but I think that not in this PR.